### DISCUSSION TITLE
Add collaborative opportunities to the curriculum

### DISCUSSION THREAD

**Discussion by rlmoser99** [TheOdinProject/curriculum#28562] [Ideas]
A common suggestion in our Discord community is to have a collaborative experience for our learners. We have organized different opportunities over the years, but none of them have been worth the amount of time and energy that it takes our volunteer team to put together.

We could add collaborative opportunities to the curriculum, so that everyone is aware of them. I'm thinking at the end of the Rails and Node course, at least for the idea to collaborate with the TOP maintainers and other Open Source projects. We could add the game jam suggestion earlier in the curriculum if we wanted.

Game Jams
https://itch.io/jams

Collaborate with the TOP maintainers
https://github.com/TheOdinProject/odin-bot-v2/issues
https://github.com/TheOdinProject/theodinproject/issues

Look for other Open Source projects
https://hacktoberfest.com/
https://goodfirstissue.dev/
https://www.firsttimersonly.com/
https://rubyforgood.org/

**Comment by silenceinspace** [comment]
I like this suggestion! Creating a meaningful pull request to a big repository and having it approved, nothing beats the feeling :D
In a way, Discord is what provides TOP students with more collaborative opportunities. At least that is where I find myself looking for collaboration of some sort when I need it. But the links to other open-source projects are pretty neat. Moreover, mentioning these at the end of Node/Ruby course feels like the perfect place.

**Comment by mao-sz** [comment]
Carrying on from the staff discussion, if the suggestions are in latter parts of the pathways (like suggested), then that looks good to me. The bot itself I don't think *needs* to wait until the end of the Node course, but if the idea is to combine the suggestion with the TOP website and other things, then that makes sense to wait until then on the JS pathway.
